Makefile

```diff
@@ -380,64 +380,58 @@ prd/checker: src/checker
 	mkdir -p $(dir $@)
 	cp $< $@
 
-src/deps.js: $(SRC_JS_FILES) .build-artefacts/python-venv
+src/deps.js: $(SRC_JS_FILES) ${PYTHON_VENV}
 	${PYTHON_CMD} node_modules/google-closure-library/closure/bin/build/depswriter.py \
 	    --root_with_prefix="src/components components" \
 	    --root_with_prefix="src/js js" \
 	    --output_file=$@
 
-src/style/app.css: src/style/app.less \
-	    src/style/print.less \
-	    src/style/offline.less \
-	    src/style/ga_bootstrap.less \
-	    src/style/ga_variables.less \
-	    $(SRC_COMPONENTS_LESS_FILES) \
-	    node_modules
-	node_modules/.bin/lessc $(LESS_PARAMETERS) $< $@
+src/style/app.css: $(SRC_LESS_FILES)
```

Comment:
since there is none less file dependent of a node module. I think we don't need it.